asm/macros/battle_script.inc

```diff
@@ -1299,13 +1299,18 @@
 	.byte 0xfe
 	.4byte \ptr
 	.endm
+	
+	.macro callnative func:req
+	.byte 0xff
+	.4byte \func
+	.endm
 
+@ various command changed to more readable macros
 	.macro metalburstdamagecalculator ptr:req
-	.byte 0xff
+	callnative BS_CalcMetalBurstDmg
 	.4byte \ptr
 	.endm
 
-@ various command changed to more readable macros
```

Comment:
I would keep this comment here to denote which macros are `various` vs which ones are `callnative`.

Reply:
done